monitoring/src/main/java/com/l2l3/monitoring/repository/EsMonitoringRepository.java

```diff
@@ -0,0 +1,22 @@
+package com.l2l3.monitoring.repository;
+
+import org.springframework.web.client.RestTemplate;
+
+import com.l2l3.monitoring.model.MonitoringData;
+
+public class EsMonitoringRepository implements MonitoringRepository {
+
+    private final String elasticsearchClusterUrl;
+    private final RestTemplate restTemplate;
+
+    public EsMonitoringRepository(RestTemplate restTemplate, String elasticsearchClusterUrl) {
+	this.restTemplate = restTemplate;
+	this.elasticsearchClusterUrl = elasticsearchClusterUrl;
+    }
+
+    @Override
+    public void save(MonitoringData data) {
+	restTemplate.postForObject(elasticsearchClusterUrl, data, MonitoringData.class);
```

Comment:
I was thinking a little. The current approach with @Async will result in 1 HTTP request for each logged MonitoringData. In certain cases this is will not be optimal. We might try to collect the incoming MonitorinDatas in a collection and then we can send all of them in one batch, (i.e.: once per every second). This would reduce the load on the elasticsearch.

please create the overload `MonitoringRepository.save(List<MonitoringData> monitoringDatas)`
create an alternate version of the MonitoringService which collects the MonitoringData entries and sends them to the MonitoringRepository in batches.